### ISSUE TITLE
Comparing two data points ?

### ISSUE THREAD

**Issue by Monox18** [leeoniya/uPlot#361]
Hello sir,

How can I compare two datapoints to show the X difference and Y difference? Say point A is selected on mouseClick and Point B is any other point on mouseHover.

Thanks!

**Comment by leeoniya** [comment]
you can get the scale value of any pixel on the graph by using the pos/val APIs:

https://github.com/leeoniya/uPlot/blob/01087cb4487e2bc7e4d544da906bd0344aa1156b/dist/uPlot.d.ts#L90-L100

you can see examples of hoe they're used in different demos:

https://github.com/leeoniya/uPlot/search?q=posToVal

**Comment by Monox18** [comment]
Thank you sir, I have been able to do what I wanted. Now the user can click on a point, see a visual feedback of the selected point with also series focus, and get the difference with any other point on mouse movement in the tooltip.

I used functions from the Draw Hooks, tooltip on closest datapoint and placement.js demos. Very helpful. And most importantly it runs very fast, reason why I chose uPlot after seeing the benching results. 

Here is a visual feedback:
![Screenshot 2020-11-23 16 36 25](https://user-images.githubusercontent.com/10236768/99983036-aa045400-2dab-11eb-80d5-8c3c1838352d.png)
